### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Kohulan**

### CODE REVIEW

**Comment by JonasSchaub** [comment]
Does the test/build fail because of a linting issue, do I see this correctly? Apart from that, it seems like all tests were successful.

**Comment by Kohulan** [comment]
Everything passed. The automated release also now works. I will fix the rest of the things then we could merge this branch.

**Comment by JonasSchaub** [comment]
Thank you!

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    branches:
+      - main
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: python
+          package-name: release-please-action
```
Comment (JonasSchaub) [inline .github/workflows/release-please.yml:18]: Shouldn't the package-name be cider-chem or sth?

Comment (Kohulan) [reply .github/workflows/release-please.yml:18]: No, it uses the package "release-please-action" to create the release. Nothing has to do with our package.

**Review by JonasSchaub — APPROVED** [review]

**Pull request merged**